workday.md

```diff
@@ -0,0 +1,30 @@
+#### When to Submit Your Hours:
+
+Submission of your hours should occur no-later than every Friday.
+
+The only exception to this is if you are working a Saturday shift, in which case Saturday Evening submission is the latest.
+
+Logging your hours before these times greatly reduces the approval process as well the chance for a late check.
```

Comment:
There's some things goin' on here. Suggested: Logging you hours before(after?) these times greatly reduces the approval proves (sounds strange though) as well as increases the chances for a late check.